api_tests/nodes/views/test_node_forks_list.py

```diff
@@ -10,8 +11,11 @@
     ForkFactory
 )
 from rest_framework import exceptions
+from website import mails
 from website.util import permissions
 
+from api.nodes.serializers import NodeForksSerializer
+mails.send_mail = mock.MagicMock()
```

Comment:
Should this use the decorator?

I've seen cases where using a patcher that is not stopped can linger beyond the test class and I don't know if mocking like this can do the same.